applications/nrf_desktop/description.rst

```diff
@@ -1131,6 +1131,11 @@ Building information
 Use the configuration with the ``release`` file suffix for the HID report rate measurement.
 Debug features, such as logging or assertions, decrease the application performance.
 
+.. note::
+  You can run the application code from RAM instead of NVM to improve performance as code execution from the RAM is generally faster than from the NVM.
+  You can configure MCUboot to use the RAM load mode to run the whole application image from RAM.
+  For details, see the :ref:`nrf_desktop_configuring_mcuboot_bootloader_ram_load` section.
```

Comment:
```suggestion
.. note::

   You can run the application code from RAM instead of NVM to improve performance as code execution from the RAM is generally faster than from the NVM.
   You can configure MCUboot to use the RAM load mode to run the whole application image from RAM.
   For details, see the :ref:`nrf_desktop_configuring_mcuboot_bootloader_ram_load` section.
```